query/security/ldap/src/main/java/org/codice/ddf/admin/ldap/commons/LdapServiceCommons.java

```diff
@@ -144,9 +144,12 @@ public LdapServiceCommons(ConfiguratorSuite configuratorSuite) {
       ldapStsConfig.put(LdapLoginServiceProperties.REALM, config.bindUserInfoField().realm());
 
       ldapStsConfig.put(
-          LdapLoginServiceProperties.USER_NAME_ATTRIBUTE,
+          LdapLoginServiceProperties.LOGIN_USER_ATTRIBUTE,
           config.settingsField().usernameAttribute());
       ldapStsConfig.put(
+          LdapLoginServiceProperties.MEMBERSHIP_USER_ATTRIBUTE,
+          config.settingsField().groupAttributeHoldingMember());
```

Comment:
This will always be null if the use case is authentication. Can we do a null check here, if null then use the usernameAttribute instead? It'd also be nice to leave a todo here to fix this along with the UI.